src/rp/util/logic.clj

```diff
@@ -0,0 +1,7 @@
+(ns rp.util.logic)
+
+(defn xor
+  [p q]
```

Comment:
If we want a function that takes multiple args and returns one when only one is truthy otherwise nil, how about naming it something like `one-and-only` or `just-one`?